mujoco_warp/_src/warp_util.py

```diff
@@ -97,6 +98,11 @@ def wrapper(*args, **kwargs):
     global _STACK
     if _STACK is None:
       return fn(*args, **kwargs)
+
+    for frame_info in inspect.stack():
+      if frame_info.function in ("capture_while", "capture_if"):
```

Comment:
@adenzler-nvidia @Kenny-Vilella note here that I'm checking the call stack and not emitting `wp.Event` if I'm inside a `capture_while` or `capture_if`.  This approach is a bit brittle as if you guys ever change the API function names, this will break.

Let me know if there's a better way of asking "Am I inside a conditional graph"